qiskit_ibm/api/rest/runtime.py

```diff
@@ -54,13 +54,19 @@ def program_job(self, job_id: str) -> 'ProgramJob':
         """
         return ProgramJob(self.session, job_id)
 
-    def list_programs(self) -> List[Dict]:
+    def list_programs(self, limit: int, offset: int) -> List[Dict]:
         """Return a list of runtime programs.
 
         Returns:
             JSON response.
         """
         url = self.get_url('programs')
+        if limit != 20 and offset == 0:
```

Comment:
not sure if this logic should be split off in its own class or maybe the `Program class`?